app-admin/scaleway-cli/scaleway-cli-1.14.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+GOLANG_PKG_IMPORTPATH="github.com/scaleway"
+GOLANG_PKG_ARCHIVEPREFIX="v"
+GOLANG_PKG_BUILDPATH="/cmd/scw"
+GOLANG_PKG_HAVE_TEST=1
+
+inherit bash-completion-r1 golang-single
+
+DESCRIPTION="Official command line tool to interact with Scaleway API"
+HOMEPAGE="https://github.com/scaleway/scaleway-cli"
+LICENSE="MIT"
+
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="zsh-completion"
+RDEPEND="zsh-completion? ( app-shells/zsh )"
+
+src_compile() {
+        golang-single_src_compile
+}
+
+src_install() {
+        golang-single_src_install
+
+        # Every Scaleway's manual mentions 'scw' command, not 'scaleway-cli'.
+        # It seems reasonable to follow their naming.
+        mv "${ED}/usr/bin/scaleway-cli" "${ED}/usr/bin/scw"
```

Comment:
You should add "|| die" to line 32.
However, instead of manually renaming the binary, you should try the eclass variable `GOLANG_PKG_OUTPUT_NAME`; it should do the same trick (unless the package resolves to some odd way to build the binary). If it doesn't work, well... we'll stick with the `mv` command.

Apart from that, the ebuild seems good to me. As soon as you fix it, I'll commit the PR.

Thanks for the ebuild! I really appreciate the contribution.

Reply:
Fixed. It seems like `GOLANG_PKG_OUTPUT_NAME` does the trick.